source/stylesheets/refills/_laptop.scss

```diff
@@ -0,0 +1,73 @@
+.laptop {
+  $lid-background: darken(#fff, 5);
+  $inset-shadow: inset 0 0px 1px transparentize(#000, 0.8);
+  $laptop-size-ratio: 1.5;
+  $bottom-width: 22em * $laptop-size-ratio;
+  $laptop-lid-width: 17em * $laptop-size-ratio;
+  $screen-height: 137px * $laptop-size-ratio;
+  $radius-outer: 15px * $laptop-size-ratio;
+  $radius-bottom: 15px * $laptop-size-ratio;
+  $padding-screen: 15px * $laptop-size-ratio;
+  $bottom-height: 15px * $laptop-size-ratio;
+  $bottom-detail-width: 3em * $laptop-size-ratio;
+  $bottom-detail-height: 4px * $laptop-size-ratio;
+  $shadow-height: 4px * $laptop-size-ratio;
+  $shadow-width: 25em * $laptop-size-ratio; 
+  $shadow-distance: -1px * $laptop-size-ratio;
+  $outlines: 1px solid #c0c0c0;
+
+  margin: auto;
+  position: relative;
+
+  .laptop-lid {
+    background: $lid-background;
+    border-radius: $radius-outer $radius-outer 0 0;
+    border: $outlines;
+    box-shadow: inset 0 2px 0 #fff;
+    margin: auto auto -1px;
+    padding: $padding-screen;
+    width: $laptop-lid-width;
+  }
+
+  .laptop-screen {
+    background: url("https://raw.githubusercontent.com/thoughtbot/refills/master/source/images/refills-screenshot2.jpg");
+    background-size: 100%;
+    border-radius: 0.2em;
+    box-shadow: inset 0 0px 1px transparentize(#000, 0.2);
+    height: $screen-height;
+    margin: auto;
+    width: auto;
+  }
+
+  .laptop-bottom {
+    background: $lid-background;
+    border-radius: 0 0 $radius-bottom $radius-bottom;
+    border: $outlines;
+    box-shadow: inset 0 2px 2px #fff;
+    height: $bottom-height;
+    margin: auto;
+    position: relative;
+    width: $bottom-width;
+
+    &:before {
+      background: #d8dbe1;
+      border-radius: 0 0 5px 5px;
+      box-shadow: 0 2px 1px #fff;
+      content: "";
+      height: $bottom-detail-height;
+      left: 50%; 
+      margin-left: -($bottom-detail-width/2);
+      position: absolute;
+      top: 0;
+      width: $bottom-detail-width;
+    }
+  }
+
+  .laptop-shadow {
+    background: transparentize(#000, 0.97);
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.